src/UCTSearch.cpp

```diff
@@ -40,6 +41,37 @@ using namespace Utils;
 
 constexpr int UCTSearch::UNLIMITED_PLAYOUTS;
 
+class OutputAnalysisData {
+public:
+    OutputAnalysisData(std::string move, int visits, int winrate, std::string pv) :
+        m_move(move), m_visits(visits), m_winrate(winrate), m_pv(pv) {};
+
+    std::string infoString(int order) const {
+        std::string tmp = "info move " + m_move + " visits " + std::to_string(m_visits) +
```

Comment:
auto tmp

Also I think the general style of the code would make this get_info_string().